analysis/import/SourceCodeParser/src/test/kotlin/de/maibornwolff/codecharta/importer/sourcecodeparser/ParserDialogTest.kt

```diff
@@ -42,7 +43,7 @@ class ParserDialogTest {
         } returns findIssues andThen defaultExcludes andThen isCompressed andThen isVerbose
         mockkStatic("com.github.kinquirer.components.ListKt")
         every {
-            KInquirer.promptListObject<OutputFormat>(any(), any())
+            KInquirer.promptListObject(any(), any<List<Choice<OutputFormat>>>(), any(), any(), any())
```

Comment:
Is it promptListObject is really called 5 times?

Reply:
This does not mean it's called 5 times, but that it has five parameters